requirements.txt

```diff
@@ -2,3 +2,5 @@ jwcrypto
 six
 python-etcd
 requests
+docker-py
+configparser; python_version < '3.4'
```

Comment:
I think we care only for 2.7 and 3.4, but it is ok to not intentionally break 3.3, just add a comment please.